src/sql/azdata.proposed.d.ts

```diff
@@ -539,6 +539,21 @@ declare module 'azdata' {
 		appendData(data: any[][]): Thenable<void>;
 	}
 
+	export interface ListViewOption {
+		/**
+		 * The optional accessibility label for the column. Default is the label for the list view option.
+		 */
+		ariaLabel?: string;
+		/**
+		 * Specify the icon for the option. The value could the path to the icon or and ADS icon defined in {@link SqlThemeIcon}.
+		 */
+		icon?: IconPath;
+		/**
+		 * Add icon padding to the list item
+		 */
+		addIconPadding?: boolean;
```

Comment:
we shouldn't need a separate property for this, if the icon is specified, then the padding should be added automatically.

Reply:
This is to add forceIconPadding. In some situations, list option may not have icons but we want them to be aligned to ones that have the icons.

Reply:
if that is the case I would prefer a different name, e.g. addIndention.

Reply:
or, we can still automatically whether any of the item in the list has icon specified, if so we add padding for all items so that they are aligned.

Reply:
I like that second option - I can't imagine that someone would want to have it not be indented in that case.